home/jobs/OMERO-server/config.xml

```diff
@@ -18,8 +18,8 @@
         </hudson.model.StringParameterDefinition>
         <hudson.model.StringParameterDefinition>
           <name>DATA_DIR</name>
-          <description>Default is $WORKSPACE/data</description>
-          <defaultValue>$WORKSPACE/data</defaultValue>
+          <description>Default is $HOME/omero-server-data</description>
+          <defaultValue>$HOME/omero-server-data</defaultValue>
```

Comment:
I am not sure how to compare the changes here with what I can see in my devspace.

Reply:
The description would only be visible in the `/configure` page of the `OMERO-server` job in jenkins.

Reply:
@joshmoore thank you, found it. Yes, this matches with what I have there too.